lib/formatters/getById.js

```diff
@@ -1,23 +1,25 @@
 'use strict';
 
-module.exports = function (rows) {
-  var tags;
+module.exports = function (chal) {
```

Comment:
It might be worth to add a jsdoc comment to describe the format of the argument chal, this will help to understand the logic of the formater:

```
Chal is an array where the length is equal of the number of tags linked to the challenge. The array has the following format:
[ anonymous {
    id: 3,
    title: 'Challenge Number 3',
    description: 'Where can I...?',
    creator_id: 3,
    org_id: 1,
    org_name: 'Apple',
    tags_id: 105,
    tags_name: 'Buildings design' },
  anonymous {
    id: 3,
    title: 'Challenge Number 3',
    description: 'Where can I...?',
    creator_id: 3,
    org_id: 1,
    org_name: 'Apple',
    tags_id: 85,
    tags_name: 'Fertiliser' },
  anonymous {
    id: 3,
    title: 'Challenge Number 3',
    description: 'Where can I...?',
    creator_id: 3,
    org_id: 1,
    org_name: 'Apple',
    tags_id: 49,
    tags_name: 'Waste to energy' } ]

```